.github/workflows/flake8.yml

```diff
@@ -0,0 +1,17 @@
+name: Flake8
+on: pull_request
+jobs:
+    flake8:
+      name: Check code with Flake8
+      runs-on: ubuntu-20.04
+      container: fedora:34
```

Comment:
do you have a specific reason to use 34 and not 36?

Reply:
This is the version that was on HW1, I assume everyone is working off that. If there's not problem upgrading and it's not going to break anything backward then np yea lets upgrade.

Reply:
After we will make sure the newer version is working without any issues we can change it